samples/notebooks/prediction/SDK_Custom_Predict_SDK_Integration.ipynb

```diff
@@ -0,0 +1,1311 @@
+    "\n",
+    "* `model.joblib` (model artifact)\n",
+    "* `preprocessor.pkl` (model artifact)\n",
+    "\n",
+    "Run the following commands to upload your files:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "ca67ee52d4d9"
+   },
+   "outputs": [],
+   "source": [
+    "%cd ..\n",
+    "!gsutil cp model_artifacts/* {BUCKET_NAME}/{MODEL_ARTIFACT_DIR}/\n",
+    "!gsutil ls {BUCKET_NAME}/{MODEL_ARTIFACT_DIR}/"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "480a1d88ecdb"
+   },
+   "source": [
+    "## Build a custom serving container using the CPR model server\n",
+    "\n",
+    "Now that the model and processor has been trained and saved, it's time to build the custom serving container. Typically building a serving container requires writing model server code. However, with the Custom Prediction Routine feature, Vertex AI Prediction has published a [model server](https://github.com/googleapis/python-aiplatform/blob/custom-prediction-routine/google/cloud/aiplatform/prediction/model_server.py) that can be used out of the box.\n",
+    "\n",
+    "A custom serving container contains the follow 3 pieces of code:\n",
+    "1. [Model server](https://github.com/googleapis/python-aiplatform/blob/custom-prediction-routine/google/cloud/aiplatform/prediction/model_server.py)\n",
+    "    * HTTP server that hosts the model\n",
+    "    * Responsible for setting up routes/ports/etc.\n",
+    "    * In this example we will use the `google.cloud.aiplatform.prediction.model_server.ModelServer` out of the box.\n",
+    "1. [Request Handler](https://github.com/googleapis/python-aiplatform/blob/custom-prediction-routine/google/cloud/aiplatform/prediction/handler.py)\n",
+    "    * Responsible for webserver aspects of handling a request, such as deserializing the request body, and serializing the reponse, setting response headers, etc.\n",
+    "    * In this example, we will use the default Handler, `google.cloud.aiplatform.prediction.handler.PredictionHandler` provided in the SDK.\n",
+    "1. [Predictor](https://github.com/googleapis/python-aiplatform/blob/custom-prediction-routine/google/cloud/aiplatform/prediction/predictor.py)\n",
+    "    * Responsible for the ML logic for processing a prediction request.\n",
+    "\n",
+    "Each of these three pieces can be customized based on the requirements of the custom container. In this example, we will only be modify the `Predictor`.\n",
+    "\n",
+    "\n",
+    "A [`Predictor`](https://github.com/googleapis/python-aiplatform/blob/custom-prediction-routine/google/cloud/aiplatform/prediction/predictor.py)must implement the following interface:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "7wj8T6yA5Ud4"
+   },
+   "outputs": [],
+   "source": [
+    "class Predictor:\n",
```

Comment:
Change this code block to markdown. I've already done so for the M2 notebook in https://github.com/googleapis/python-aiplatform/pull/1045.

Reply:
Done.